lib/src/main/kotlin/com/swmansion/starknet/data/serializers/IntWrapperSerializer.kt

```diff
@@ -0,0 +1,21 @@
+package com.swmansion.starknet.data.serializers
+
+import com.swmansion.starknet.data.types.IntWrapper
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.descriptors.PrimitiveKind
+import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+
+object IntWrapperSerializer : KSerializer<IntWrapper> {
```

Comment:
not sure about the name tbh

Reply:
Maybe `HttpBatchRequestString`? It's less general then current `IntWrapper`.

Reply:
This name is not intuitive, let's remember this is a type that we will return no matter if someone uses batched request or not. Perhaps something like `StarknetInt` would be a bit better?

Reply:
Correct, `StarknetInt` seems better and is suitable to all types of requests.